code/index.html

```diff
@@ -1,30 +1,89 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
 
-    <!-- Signup form -->
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Project 3</title>
+  <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/css?family=IBM+Plex+Mono&display=swap" rel="stylesheet">
+  <!-- dont forget to add a css file and link it here! -->
+</head>
+
+<body>
+  <header>
+    <div class="hero">
+      <div class="logo"></div>
+      <h1>Ceramics for everyone</h1>
+      <h2>One for all and all for pottery! </h2>
+    </div>
+  </header>
+  <!-- Signup form -->
+
+  <section class="container">
+
+    <section class='signup'>
```

Comment:
You can tell here that both `'` and `"` around the class name works! How you want the syntax to be is up to you - but choose one and stick to it throughout the project. I can see that you mostly use `"` as syntax so then maybe this would be changed to that aswell!
```suggestion
    <section class="signup">
```